stringfuzz/ast.py

```diff
@@ -12,9 +12,11 @@
     'StringLitNode',
     'SortNode',
     'SettingNode',
+    'MetaDataNode',
```

Comment:
These changes will need to be propagated to the stringfuzzx filter that removes these kind of things. Right?

Reply:
Right! I'll add that fix.